apps/web/modules/survey/components/question-form-input/components/recall-wrapper.tsx

```
@@ -63,6 +63,10 @@ export const RecallWrapper = ({
   const [renderedText, setRenderedText] = useState<JSX.Element[]>([]);
   const fallbackInputRef = useRef<HTMLInputElement>(null);
 
+  const hasRecallItems = useMemo(() => {
+    return recallItems.length > 0 || (value && value.includes("recall:"));
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)